CHANGELOG.md

```diff
@@ -29,6 +51,10 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
 
 - Improved performance of mobilecoind-dev-faucet for better load testing ([#2955])
 
+[#2988]: https://github.com/mobilecoinfoundation/mobilecoin/pull/2988
+[#2952]: https://github.com/mobilecoinfoundation/mobilecoin/pull/2952
+[#2955]: https://github.com/mobilecoinfoundation/mobilecoin/pull/2955
+
```

Comment:
```suggestion
[#3062]: https://github.com/mobilecoinfoundation/mobilecoin/pull/3062

```

Reply:
Where is this referenced?

Reply:
Sorry. I'm not sure what you mean. I thought it should be included because it was part of the dev faucet changes.